mediaire_toolbox/transaction_db/migrations.py

```diff
@@ -14,5 +14,13 @@
     ],
     4: [
         "ALTER TABLE transactions ADD COLUMN task_cancelled INT DEFAULT 0",
+    ],
+    5: [
+        "ALTER TABLE transactions ADD COLUMN status TEXT",
+        "ALTER TABLE transactions ADD COLUMN institution TEXT",
+        "ALTER TABLE transactions ADD COLUMN sequences TEXT",
+        "UPDATE transactions SET status = 'sent_to_pacs' WHERE processing_state = 'send_to_pacs'",
+        "UPDATE transactions SET status = 'reviewed' WHERE processing_state != 'send_to_pacs'"
```

Comment:
Should the default for transactions not sent to pacs be `reviewed` or `unseen`?

Reply:
yeah, good question. Maybe unseen is better since they really didn't click on the new level 2 on any of the old ones. Will change then.